homeworks/yehor.honcharenko_NOcode-developer/GamesProject/About.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Game Project</title>
+        <link rel="stylesheet" href="./Style/About.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <section class="body__wrapper">
+            <header class="nav">
+                <nav class="nav__container">
+                    <img class="logoTop" src="./Images/logo-ma.svg" width="64px" height="64px" alt="logotype master of code academy">
+                    <ul class="nav__ul">
+                        <li class="nav__li">
+                            <a class="nav_link" href="Cards.html">Games</a>
+                        </li>
+                        <li class="nav__li">
+                            <a class="nav_link" href="#">About</a>
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+            <main class="content">
+                <section class="content__container">
+                    <article class="content__img">
+                        <img src="./Images/image-about1.svg" alt="image about us" style="width: 100%;">
+                    </article>
+                    <article class="content__about">
+                        <article class="about__header">
+                            <h1 class="header__title">About me</h1>
```

Comment:
this is an element `title` of block `header`? But where is the block `header`?
Maybe it is and element `title` of block `about-text`? Same for 
- about__text
- text__paragraph-black
- link__paragraph-black

Reply:
I attempted to fix this, but I'm uncertain if it's the correct solution

Reply:
OK, you've fixed almost all, well done!